modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/js/diff-view.js

```diff
@@ -0,0 +1,18 @@
+$("#diff-view-version li").on('mouseover',function(event){
```

Comment:
Thanks @splinter , I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/0f0795aa166ee07f2ea2ab7bd0b9bccfdde34e90)